draco/pipelines/double_lstm/double_lstm.json

```diff
@@ -34,17 +34,13 @@
             "key": "timestamp"
         },
         "mlprimitives.custom.timeseries_preprocessing.cutoff_window_sequences#1": {
-            "window_size": 72,
+            "window_size": 24,
```

Comment:
Is this value selected based on a certain strategy?

Reply:
It's only a template, and the data we have is resampled by 1 hour, so I made the window size 24 to express 1 day. The reason I made this change is to have faster testing.